.github/workflows/weekly-cron-tests.yaml

```diff
@@ -1,94 +1,83 @@
-name: Weekly CNI tests
+name: Weekly Cron tests
 
 on:
   schedule:
-    - cron: "0 16 * * 3"  # every Wednesday
+    - cron: "0 16 * * 3" # every Wednesday
 
 permissions:
   contents: read
 
 jobs:
-  # Run weekly tests on self-hosted runner
   weekly-cron:
-    runs-on: [self-hosted, linux, x64]
+    runs-on: ubuntu-latest
     steps:
-      - name: Set up Go 1.18
-        uses: actions/setup-go@v2
+      - name: Checkout latest commit in the PR
+        uses: actions/checkout@v3
+      - name: Set up Docker QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Set up Go
+        uses: actions/setup-go@v3
         with:
-          go-version: '1.18'
-        id: go
-
-      - name: Check out code into the Go module directory
-        uses: actions/checkout@v2
-
-      - name: Clean up docker
-        run:  docker system prune -a --volumes -f
-
-      - name: Clean up stale docker images
-        run:  docker image prune -f
-
-      - name: Get dependencies
+          go-version: "1.18"
+      - name: Set up tools
         run: |
-          go install golang.org/x/lint/golint@latest
-          go install golang.org/x/tools/cmd/goimports@latest
-
+          go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo@latest
+          curl --silent --location "https://github.com/weaveworks/eksctl/releases/latest/download/eksctl_$(uname -s)_amd64.tar.gz" | tar xz -C /tmp
+          sudo mv /tmp/eksctl /usr/local/bin/
+      - name: Set up AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: ${{ secrets.OSS_TEST_ROLE_ARN }}
+          role-duration-seconds: 7200
+          aws-region: ${{ secrets.AWS_DEFAULT_REGION }}
       - name: Run perf tests
         env:
           DISABLE_PROMPT: true
+          S3_BUCKET_CREATE: false
+          S3_BUCKET_NAME: ${{ secrets.S3_BUCKET_NAME }}
           ROLE_CREATE: false
           ROLE_ARN: ${{ secrets.ROLE_ARN }}
-          MNG_ROLE_ARN: ${{ secrets.MNG_ROLE_ARN }}
-          RUN_PERFORMANCE_TESTS: true
           PERFORMANCE_TEST_S3_BUCKET_NAME: cni-performance-tests
-          AWS_DEFAULT_REGION: ${{ secrets.AWS_DEFAULT_REGION }}
+          RUN_PERFORMANCE_TESTS: true
           RUN_TESTER_LB_ADDONS: true
-          S3_BUCKET_CREATE: false
-          S3_BUCKET_NAME: ${{ secrets.S3_BUCKET_NAME }}
         run: |
           ./scripts/run-integration-tests.sh
```

Comment:
This script is already set above to run nightly. Maybe we should run `run-cni-release-tests` nightly instead?

Reply:
This is the performance test - "RUN_PERFORMANCE_TESTS: true"

Reply:
We run performance, bottlerocket and calico tests weekly.

Reply:
Ah, I see now that the env vars determine which tests run, thank you for clarifying. Resolving this.